psydac/api/ast/linalg_kernels.py

```diff
@@ -1,13 +1,14 @@
+from pyccel.decorators import template
```

Comment:
Please add an empty line after the imports

Reply:
[86b06e9](https://github.com/pyccel/psydac/pull/273/commits/86b06e97fda7a028f2d6ec3f23b6c81d44eaf696)